README.md

```diff
@@ -1,13 +1,19 @@
+# Description
+This simple code is a fork of great inital work made by nstansby. I've added tew tweaks so I can use the code for my little project
+* possibility to set the min and max limits for the counter
+* possibility to initiate the counter with an arbitrary value
+* capability of setting the value of the counter at any time
+
 # rpi-rotary-encoder-python
 I had a quick look for some clean code to listen to a rotary encoder attached to a Raspberry Pi, and didn't find any... so I rolled my own.
 
 ## Usage
 
-First of all, you're going to need to instantiate an Encoder instance, and give it the two GPIO pins to listen on:
+First of all, you're going to need to instantiate an Encoder instance, and give it the two GPIO pins to listen on as well as additional start parameters - limits (min and max) and start counter value:
 
 ```python
 from encoder import Encoder
-e1 = Encoder(26, 19)
+e1 = Encoder(26, 19, 0, 100, 20)
```

Comment:
I'd prefer to use the explicit notation `e1 = Encoder(26, 19, minValue=0, maxValue=100, initialValue=20)`.  Once it becomes a long string of unnamed integers, it's pretty much impossible for someone reading the code to understand without going and looking at the function declaration.  Yeah, I know you could argue that's true for the first two arguments, and we're all using IDEs and can trivially jump to the declaration, but there's just something about `e1 = Encoder(26, 19, 0, 100, 20)` that offends my sensibilities!